docs/source/datasets/index.rst

```diff
@@ -461,4 +461,148 @@ The following columns are included in these datasets:
 Tax form: 1040
 --------------
 As with data collected from W-2 and 1099 forms, Pseudopeople will also enable the simulation of administrative records from 1040 forms, which are
-also reported to the IRS on an annual basis. This feature has not yet been implemented, so please stay tuned for more information! 
+also reported to the IRS on an annual basis. To find out more about the 1040 tax form, visit the `IRS information page <https://www.irs.gov/instructions/i1040gi>`_.
+
+Generate 1040 data with pseudopeople.generate_taxes_1040.
+
+The following columns are included in these datasets:
+
+.. list-table:: **Dataset columns**
+   :header-rows: 1
+
+   * - Attribute Name
+     - Column Name
+     - Notes
+   * - Unique simulant ID
+     - :code:`simulant_id`
+     - Not affected by noise functions; intended use is "ground truth" for testing and validation.
+   * - Unique household ID
+     - :code:`household_id`
+     - Not affected by noise functions; intended use is "ground truth" for testing and validation; consistent across all
+       datasets. 
+   * - First name
+     - :code:`first_name`
+     - 
+   * - Middle initial
+     - :code:`middle_initial`
+     - 
+   * - Last name
+     - :code:`last_name`
+     - 
+   * - Age
+     - :code:`age`  
+     - Rounded down to an integer.
+   * - Date of birth
+     - :code:`date_of_birth`
+     - Formatted as YYYY-MM-DD.
```

Comment:
double check that this is the format we want; i could imagine MM/DD/YYYY being more appropriate, but I don't mind whatever we choose.

Reply:
Age and DOB [should not appear at all in tax forms](https://github.com/ihmeuw/vivarium_research/pull/1235). The "Age" and "Date of birth" rows should be removed.

Reply:
According to the [pseudopeople docs](https://pseudopeople.readthedocs.io/en/latest/datasets/index.html), all of our other datasets have DOB in YYYY-MM-DD format as well (Census, ACS, CPS, WIC, SSA). @zmbc does that seem accurate?